### ISSUE TITLE
uint tween overflow

### ISSUE THREAD

**Issue by SergeyRomanko** [Demigiant/dotween#71]
Hey!
Could you please take a look at this code.

```

using UnityEngine;
using System.Collections;
using DG.Tweening;

public class TweenTest : MonoBehaviour {

    private uint test = 1000;
    public uint getTest(){
        return test;
    }

    public void setTest(uint value){
        test = value;
        Debug.Log("test>> " + test);
    }


    // Use this for initialization
    void Start () {
        uint to = 0;
        DOTween.To(getTest, setTest, to, 2.0f);
    }

    // Update is called once per frame
    void Update () {

    }
}

```

Output

```

...
test>> 4294574848
test>> 4294960384
test>> 0

```

**Comment by Demigiant** [comment]
Uh, damn. When I made the uint plugin I didn't consider that we can't have negative uints, and thus storing a changeValue (which is the tween's endValue minus the startValue) for decreasing uints won't work. I'll have to ponder about this a little. Can you use ints in the meantime? Also, any suggestion?

**Comment by SergeyRomanko** [comment]
Sure, I can use ints for some time, it is not a big deal.

Regarding suggestion, you may introduce new variable for storing sign.

```

bool isNegative = true;
uint changeValue = 1000;

if(isNegative){
    newValue = startValue-changeValue*tweenProgress;
}else{
    newValue = startValue+changeValue*tweenProgress;
}
```

But this is just a guess. I think your architecture is much more complicated than this.

**Comment by Demigiant** [comment]
Fixed! :) You can get the new version (1.1.300) [here](http://dotween.demigiant.com/download.php).
